### DISCUSSION TITLE
how could we define the mutualTLS ?

### DISCUSSION THREAD

**Discussion by RSH17** [OAI/OpenAPI-Specification#3060] [Q&A]
> Under Open API specification 3.1.0 version, there is a defined Security Scheme type called mutualTLS. how could we define this type using code? 
could you please give an example of this type?

**Comment by rafalkrupinski** [comment]
I think you just define client cert&key and server certificate
Python example with [httpx](https://www.python-httpx.org/advanced/#ssl-certificates)

```python
import httpx

cert = ("path/to/client.pem", "path/to/client.key", "password")
r = httpx.get("https://example.org", cert=cert, verify="path/to/ca.pem")
```

**Comment by mcrobbj-SilverBullet** [comment]
I havent seen an example of this define in yaml can someone share, and also an example at the path level

**Reply by commonism** [reply]
https://github.com/OAI/OpenAPI-Specification/pull/2625/files
https://github.com/commonism/aiopenapi3/blob/master/tests/tls_test.py#L94

**Comment by handrews** [comment]
This seems to have been resolved, closing.
